include/rawpheno.backup.form.inc

```diff
@@ -165,7 +165,10 @@ function rawpheno_backup($form, &$form_state, $project_id = NULL, $command = 'de
         '#file_upload_max_size' => '10M',
         '#upload_location' => 'public://',
         '#upload_event' => 'manual',
-        '#upload_validators' => array('file_validate_extensions' => array('xlsx xls')),
+        '#upload_validators' => array(
+          'file_validate_extensions' => array('xlsx xls'),
+          'rawpheno_validate_spreadsheet_file' => array($form_state),
```

Comment:
Lacey, I need to do more test on this. It appears that both upload and backup freeze when spreadsheet has no measurements tab.